ansible/druid-ingestion.yml

```diff
@@ -1,6 +1,56 @@
 ---
+- hosts: rollup-overlord
+  become: yes
+  pre_tasks:
```

Comment:
Move this entire block as a separate play running against `localhost`, so we can remove duplication of code.
```
- hosts: localhost
  tasks:
   - name: set_fact
     set_fact:
       single_cluster: true
     tags: populate_var

   - name: check for hosts
     set_fact:
       single_cluster: false
     when: "'rollup-overlord' in groups"
     tags: populate_var
```

Reply:
This variable single_cluster is not available for next play. As we are setting this variable for loacalhost host group, it is not available for next play when we run against rollup-overload or raw-overlord. That is why I added this set_fact in each play.
@keshavprasadms

Reply:
@Kaali09 Can you try to use this in the when condition?
`{{ hostvars['localhost']['single_cluster'] }}`

Reply:
thanks . it worked without curly braces.